tests/components/BidHistory.test.tsx

```diff
@@ -138,7 +138,8 @@ describe("AuctionInfo", () => {
       );
 
       const wonArea = await screen.findByText("won the auction");
-      expect(wonArea.parentElement?.innerHTML).toContain("May 14, 10:23 PM");
+      
+      expect(wonArea.parentElement?.parentElement?.parentElement?.innerHTML).toContain("May 14, 10:23 PM");
```

Comment:
Can just do `await screen.findByText("May 14, 10:23 PM");` instead.

Reply:
Changed to `await screen.findAllByText("May 14, 10:23 PM");` as there are two instances of the datestamp.